docker_config/docker_command.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+echo "Pull changes"
+cd /GraphSpace
+git stash 
+git pull
+
+echo "Copy configurations"
+yes | cp -rf /GraphSpace/docker_config/elasticsearch/elasticsearch.yml /elasticsearch/config/elasticsearch.yml
+yes | cp -rf /GraphSpace/docker_config/redis/redis.conf /redis/redis.conf
+yes | cp -rf /GraphSpace/docker_config/kafka/server.properties /kafka/config/server.properties
+#yes | cp /GraphSpace/docker_config/supervisord/graphspace.conf /etc/supervisor/conf.d/
```

Comment:
Not needed. 
`supervisord` directly uses the settings from `/GraphSpace/docker_config/supervisord/graphspace.conf` (see line L#66)